### DISCUSSION TITLE
关于评论样式的问题 || Questions about comment style

### DISCUSSION THREAD

**Discussion by ReverseSacle** [walinejs/waline#2392] [General]
当我想修改评论的加载动画时，由于HTML布局是固定的，无法自定义动画样式。想问下如何修改源码来自定义HTML布局？
另外关于RecentComments这个api，我能否获取n条父评论？最近评论获取的sql逻辑是按InsertAt属性来获取的吧？我看了下api返回的json数据，有没有什么方法能返回n条【"comment"与"orig"一致】的父评论？

---
When I want to modify the loading animation of comments, since the HTML layout is fixed, the animation style cannot be customized. I would like to ask how to modify the source code to customize the HTML layout?
Also regarding the RecentComments API, can I get n parent comments? The sql logic obtained from recent comments is obtained based on the InsertAt attribute, right? I looked at the json data returned by the api. Is there any way to return n parent comments ["comment" is consistent with "orig"]?

**Comment by lizheming** [comment]
1. HTML 布局不好自定义，有诉求可以自己使用 `@waline/api` 模块调用 API 自己去实现前端组件
2. API 如果不满足你的诉求，可以通过自定义插件自定义 API 来返回数据 https://waline.js.org/reference/server/plugin.html

**Reply by ReverseSacle** [reply]
你这API文档的例子也太少了，我找不到相关的内容。
有没有什么方法能在HTML布局构建时，选定标签来添加子标签之类的或者HTML布局构建完后，选定标签之类的？
另外，你这服务端API能不能【返回单条评论】或【n条评论】或【返回最近评论加上评论索引偏移量】之类的？

---
There are too few examples in your API documentation, and I can't find relevant content.
Is there any way to select tags to add sub-tags when building the HTML layout or select tags after the HTML layout is built?
In addition, can your server-side API [return a single comment] or [n comments] or [return the most recent comment plus the comment index offset] or the like?
